AGENTS.md

```diff
@@ -0,0 +1,59 @@
+# Cockpit starter-kit - Technical Overview
+
+This document provides a technical overview for applications or plugins that integrate into Cockpit and are prepared with the starter-kit. Is is human readable but intended to give AI context to follow the project standards.
+
+## Project Structure
+
+- **Upstream Website**: [Cockpit Project](https://cockpit-project.org/)
+- **Upstream Code Repository**: [Cockpit Project Git](https://github.com/cockpit-project/)
```

Comment:
This doesn't belong here, but into README. Also, a forked project usually does not live under the cockpit-project org.